entry/streamer-entry

```diff
@@ -0,0 +1,211 @@
+#!/bin/bash
+
+SELF=$(basename $0)
+
+function err() {
+  echo "ERR $SELF $$ ${FUNCNAME[1]}: $@" >&2
+}
+
+function dbg() {
+  echo "DBG $SELF $$ ${FUNCNAME[1]}: $@" >&2
+}
+
+function main() {
+  local uid=0
+  local w=${K8S_ENV_DISPLAY_RESOLUTION_X:-1280}
+  local h=${K8S_ENV_DISPLAY_RESOLUTION_Y:-720}
+  local y=${ICR_ON:-n}
+  local user_count=${K8S_ENV_USER_COUNT_PER_ANDROID:-1}
+  local dev=${DEVICE:-/dev/dri/renderD128}
+  local cqp=${K8S_ENV_FORCE_CQP:-false}
+
+  local help=$(
+    cat <<EOF
+Usage: $SELF [-i <id>] [-u <uid>]  [-h] <url>
+  Streamer provide streaming service between AIC and WebRTC client
+
+  -i <id>: instance ID, default: $id
+  -u <uid>: user id, default: 0
+  -l:      enable measuring latency
+  -h:      print the usage message
+  <ip>:    ip to signaling server in format
+
+EOF
+  )
+  local wdir="/opt/workdir"
+  local k8s=${K8S_ENV:-"false"}
+  local k8s_stateless=${K8S_ENV_STATELESS:-"false"}
+  local dev=${DEVICE:-/dev/dri/renderD128}
+  local id="$(id)"
+  local measure_latency="false"
+  local privileged_enable=${ENABLE_ENV_PRIVILEGED:-"false"}
+  local hevc_enable=${HEVC_ENABLE:-"false"}
+  local icr_start_immediately=${ENV_ICR_START_IMMEDIATELY:-"false"}
+
+  while getopts ":i:u:lh" opt; do
+    case $opt in
+    i) id=$OPTARG ;;
+    u) uid=$OPTARG;;
+    l) measure_latency="true";;
+    h) echo "$help" && exit ;;
+    *) echo "no such option: $opt" && exit 1 ;;
+    esac
+  done
+  shift $((OPTIND - 1))
+
+  if [[ -n "$1" ]]; then
+    ip="$1"
+  else
+    err "signaling server ip is required"
+    echo "$help"
+    exit 1
+  fi
+
+  dbg "id is $id"
+  dbg "ips is $ips"
+  dbg "port is $port"
+  dbg "measure_latency is $measure_latency"
+
+  local ips=$ip
+  local peerid=$[$user_count*$id+$uid]
+  local serverid=s$peerid
+  local clientid=c$peerid
+  local coturn_ip=${COTURN_IP:-"$ip"}
+  local ice_port_begin=${ICE_PORT_BEGIN:-"50000"}
+  local ice_port_end=${ICE_PORT_END:-"50999"}
+
+  local ipc files
+  if [[ "$k8s" == "true" ]]; then
+    ipc="/conn"
+    wdir="/conn"
+    nid=0
+    ips=${K8S_ENV_OWT_SERVER_IP:-owt}
+    port=${K8S_ENV_OWT_SERVER_PORT:-8095}
+    portOpt=" -p $port"
+    k8sOpt="--k8s 1"
+    local GPU_NODES=${GPU_NODES:-$(find /dev/dri -iname "renderD*" | wc | awk '{print $1}')}
```

Comment:
Why not just use `wc -l` instead of `awk`?

And this does not check whether given GPU is Intel one.

You could use script from here: https://github.com/intel/intel-device-plugins-for-kubernetes/blob/main/cmd/gpu_plugin/README.md#issues-with-media-workloads-on-multi-gpu-setups